lib/stytch/b2b_sessions.rb

```diff
@@ -13,14 +13,34 @@
 
 module StytchB2B
   class Sessions
+    class RevokeRequestOptions
+      # Optional authorization object.
+      # Pass in an active Stytch Member session token or session JWT and the request
+      # will be run using that member's permissions.
+      attr_accessor :authorization
+
+      def initialize(
+        authorization: nil
+      )
+        @authorization = authorization
+      end
+
+      def to_headers
+        headers = {}
+        headers.merge!(@authorization.to_headers) if authorization
+        headers
+      end
+    end
+
     include Stytch::RequestHelper
 
-    def initialize(connection, project_id, policy_cache)
+    def initialize(connection, project_id, is_b2b_client, policy_cache)
```

Comment:
This feels *slightly* weird to me since the `is_b2b_client` param is unused (I think we could've been smarter about deciding whether a given class _wants_ this param and limited to "hybrid" classes like M2M), but it's not hurting anything, so I think it's fine.

Reply:
Fixed in [e1fb37b](https://github.com/stytchauth/stytch-ruby/pull/130/commits/e1fb37bfaf85577eb8774257c5d4649533576af9)